lp-content-analysis/src/main/java/eu/learnpad/ca/impl/BridgeImpl.java

```diff
@@ -209,6 +210,14 @@ public String getStatusCollaborativeContentVerifications(@PathParam("idAnnotated
 		}
 	}
 
+	@Path("/validatecollaborativecontent/{idAnnotatedCollaborativeContentAnalysis:\\d+}/view")
+	@Override
+	public URL getCollaborativeContentVerificationsView(String contentID)
```

Comment:
Maybe use `VerificationID` for the name because `contentID` may be confuse with the `documentID`.